SECURITY.md

```diff
@@ -0,0 +1,31 @@
+# Critical bugs and security issues 💥
+
+If you're here because you're trying to figure out how to notify us of a security issue, go to [Discord](https://discord.gg/VNyAnw7e), and alert:
+
+
+- Ethan Frey `ethanfrey#9693`
+- elsehow `elsehow#3115`
+
+Please avoid opening public issues on GitHub that contain information about a potential security vulnerability as this makes it difficult to reduce the impact and harm of valid security issues.
+
+### Coordinated Vulnerability Disclosure Policy
+
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading services built on top of cw-plus.
+
+### Vulnerability Disclosure Process
```

Comment:
The process sounds reasonable, but I would like to discuss it with the rest of the Confio team first